all.json

```diff
@@ -1258,6 +1260,7 @@
     "mywalletbrowser.com",
     "mywalletconnector.com",
     "mywalletdapps.com",
+    "mywalletdappsvalidate.weebly.com",
```

Comment:
Not adding `weebly.com` since it will block the whole domain. Hopefully this is the right approach?

Reply:
100%